### PR TITLE
Fix Extension Docs

### PR DESCRIPTION
**Pull request by Nokel81**
- fix spelling
- add protocol handler to tree view
- make all lines a single sentance

Signed-off-by: Sebastian Malton <sebastian@malton.name>

Will merge this for the 4.2.0 release but don't want to block the rc.2 release on this.

### CODE REVIEW

**Review by ixrock — DISMISSED** [review]

File: docs/extensions/testing-and-publishing/testing.md [hunk 1]
```diff
@@ -41,19 +44,23 @@ test("click called navigate()", () => {
...
+There are more example tests in the generator's [template](https://github.com/lensapp/generator-lens-ext/tree/main/generators/app/templates/ext-ts/components).
+Extend your tests based on the examples.
 
 ## Main Process Unit Testing
 
 Code in the extension main process are just normal JavaScript files that has access to extension api, you can write unit tests using any testing framework.
 
-If you are using the [Yeoman Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) to scaffold your extension project. The testing environment [Jest](https://jestjs.io/) are setup for you. Just use  `npm start` or `yarn test` to run the tests.
+If you are using the [Yeoman Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) to scaffold your extension project.
+The testing environment [Jest](https://jestjs.io/) are setup for you.
```
Comment (jim-docker) [inline docs/extensions/testing-and-publishing/testing.md]:
```suggestion
If you are using the [Yeoman Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) to scaffold your extension project then the [Jest](https://jestjs.io/) testing environment is set up for you.
```

File: docs/extensions/testing-and-publishing/testing.md [hunk 2]
```diff
@@ -41,19 +44,23 @@ test("click called navigate()", () => {
 
 In the example we used [React Testing Library](https://github.com/testing-library/react-testing-library) but any React testing framework can be used to test renderer process UI components.
 
-There are more example tests in the generator's [template](https://github.com/lensapp/generator-lens-ext/tree/main/generators/app/templates/ext-ts/components). Extend your tests based on the examples.
+There are more example tests in the generator's [template](https://github.com/lensapp/generator-lens-ext/tree/main/generators/app/templates/ext-ts/components).
+Extend your tests based on the examples.
 
 ## Main Process Unit Testing
 
 Code in the extension main process are just normal JavaScript files that has access to extension api, you can write unit tests using any testing framework.
```
Comment (jim-docker) [inline docs/extensions/testing-and-publishing/testing.md]:
```suggestion
Code in the extension's main process consists of normal JavaScript files that have access to extension api, you can write unit tests using any testing framework.
```

File: docs/extensions/testing-and-publishing/testing.md [hunk 3]
```diff
@@ -2,9 +2,12 @@
 
 ## Renderer Process Unit Testing
 
-UI components in extension renderer process are based on React/ReactDOM. These components can be tested by popular React testing tools like [React Testing Library](https://github.com/testing-library/react-testing-library).
+UI components in extension renderer process are based on React/ReactDOM.
```
Comment (jim-docker) [inline docs/extensions/testing-and-publishing/testing.md]:
```suggestion
UI components in the extension's renderer process are based on React/ReactDOM.
```

File: docs/extensions/testing-and-publishing/testing.md [hunk 4]
```diff
@@ -2,9 +2,12 @@
...
+These components can be tested by popular React testing tools like [React Testing Library](https://github.com/testing-library/react-testing-library).
 
-If you are using the [Yeoman Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) to scaffold extension project. The testing environment for render process are already setup for you. Just use `npm start` or `yarn test` to run the tests.
+If you are using the [Yeoman Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) to scaffold extension project.
+The testing environment for render process are already setup for you.
```
Comment (jim-docker) [inline docs/extensions/testing-and-publishing/testing.md]:
```suggestion
If you are using the [Yeoman Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) to scaffold your extension project then the testing environment for the renderer process is already set up for you.
```

File: docs/extensions/testing-and-publishing/testing.md [hunk 5]
```diff
@@ -2,11 +2,10 @@
 
 ## Renderer Process Unit Testing
 
-UI components in extension renderer process are based on React/ReactDOM.
+UI components in the extension's renderer process are based on React/ReactDOM.
 These components can be tested by popular React testing tools like [React Testing Library](https://github.com/testing-library/react-testing-library).
 
-If you are using the [Yeoman Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) to scaffold extension project.
-The testing environment for render process are already setup for you.
+If you are using the [Yeoman Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) to scaffold extension project then the testing environment for render process is already setup for you.
```
Comment (jim-docker) [inline docs/extensions/testing-and-publishing/testing.md]:
```suggestion
If you are using the [Yeoman Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) to scaffold extension project then the testing environment for render process is already set up for you.
```

File: docs/extensions/testing-and-publishing/testing.md [hunk 6]
```diff
@@ -49,10 +48,9 @@ Extend your tests based on the examples.
 
 ## Main Process Unit Testing
 
-Code in the extension main process are just normal JavaScript files that has access to extension api, you can write unit tests using any testing framework.
+Code in the extension's main process consists of normal JavaScript files that have access to extension api, you can write unit tests using any testing framework.
 
-If you are using the [Yeoman Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) to scaffold your extension project.
-The testing environment [Jest](https://jestjs.io/) are setup for you.
+If you are using the [Yeoman Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) to scaffold your extension project then the [Jest](https://jestjs.io/) is setup for you.
```
Comment (jim-docker) [inline docs/extensions/testing-and-publishing/testing.md]:
```suggestion
If you are using the [Yeoman Lens Extension Generator](https://github.com/lensapp/generator-lens-ext) to scaffold your extension project then the [Jest](https://jestjs.io/) testing environment is set up for you.
```

**Review by jim-docker — COMMENTED** [review]
You didn't take my suggestions verbatim :)

**Review by jim-docker — DISMISSED** [review]

**Review by jim-docker — APPROVED** [review]

**Review by jakolehm — APPROVED** [review]

**Pull request merged**